Dockerfile-ssl

```diff
@@ -0,0 +1,29 @@
+#
```

Comment:
Why add a new file, rather than altering the existing one?

Reply:
Presumably users of this exporter will want either the ssl or the non-ssl version. 
I didnt see a reasonable way to provide a single Dockerfile that supported both alternatives.

Reply:
Why not just provide ssl? I presume it'd work for both.

Reply:
It would work for both, but the ubuntu base image is about 180MB where this alpine base is about 5MB, so if you don't need SSL, the alpine build is preferred.  Also, only old versions of alpine and ubuntu currently work with NRPE's ciphers.  

Perhaps we should rename the ssl version to the default 'Dockerfile' and provide the -alpine variant  for 
those that don't need ssl, want a faster & smaller build, and would prefer more up-to-date software. 
While I cant imagine running NRPE in plain text, If I didnt need ssl, I would be using the -alpine variant. 

I've made further changes to the ssl build file to provide a multi-stage build yielding a  19MB image.  

Please advise your desires on the rename and multi-stage build.

Reply:
no - file was deleted (renamed) in last commit; review comments remain.